### ISSUE TITLE
Add confirmation modal for "Remove wallet" action

### ISSUE THREAD

**Issue by thatStrangeGuyThorchain** [thorchain/asgardex-electron#1335]
users always may miss-click to the `remove` button so with that we need to ask him if he really wants to do it

**Comment by warrenshen** [comment]
@thatStrangeGuyThorchain is there more than one "Remove wallet" action? I checked the code and ran through the app, the "Remove wallet" action in the screenshot below has a confirmation modal. IMO the copy in the modal could use some work: change "Remove wallet" to "Are you sure you want to remove your wallet? You cannot undo this action and will have to re-create your wallet from your seed phrase." or something like that

<img width="1592" alt="Screen Shot 2021-05-30 at 9 39 06 PM" src="https://user-images.githubusercontent.com/8787796/120140482-81607b00-c18f-11eb-9362-f98a4f29a832.png">

**Comment by veado** [comment]
> IMO the copy in the modal could use some work: change "Remove wallet" to "Are you sure you want to remove your wallet? You cannot undo this action and will have to re-create your wallet from your seed phrase." or something like that

@warrenshen Nice feedback, please add such description into the modal. All texts for wallets are defined in `src/renderer/i18n/{lang}/wallet.ts`. For languages you can't do add translations, just add EN texts suffixed with {LANG}, for example:

```
// EN
'wallet.action.remove.title': 'Are you sure you want to remove your wallet?', // previously 'wallet.action.remove': 'Remove wallet'
'wallet.action.remove.description': 'You cannot undo this action and will have to re-create your wallet from your seed phrase.'

// FR
'wallet.action.remove.title': 'Are you sure you want to remove your wallet? - FR', // previously 'wallet.action.remove': 'Remove wallet'
'wallet.action.remove.description': 'You cannot undo this action and will have to re-create your wallet from your seed phrase. - FR'
```

Also it would be nice to style such modal (it's currently just a standard `antd` modal). We already have different modals components in `src/renderer/components/modal` or just create a another (better styled) component for this.

Thanks!

**Comment by warrenshen** [comment]
@veado sounds good, will do

**Comment by veado** [comment]
Resolved w/ #1479
